cmd/controller/certificates.go

```diff
@@ -64,7 +64,7 @@ func (c *Certificates) Init(ctx context.Context) error {
 	}
 	c.CACert = string(cert)
 	// Changing the URL here also requires changes in the "k0s kubeconfig admin" subcommand.
-	kubeConfigAPIUrl := fmt.Sprintf("https://localhost:%d", c.ClusterSpec.API.Port)
+	kubeConfigAPIUrl := fmt.Sprintf("https://%s:%d", c.ClusterSpec.API.Address, c.ClusterSpec.API.Port)
```

Comment:
Ideally, this would use `net.JoinHostPort`. I think it'll break for IPv6 addresses otherwise.

Reply:
makes sense, done